src/lib/mina_compile_config/mina_compile_config.ml

```diff
@@ -68,3 +68,5 @@ let max_event_elements = 100
 let max_action_elements = 100
 
 [%%inject "network_id", network]
+
+let zkapp_cmd_limit = None
```

Comment:
@nholland94 I know this isn't correct, I just put something in so it would compile. I tried putting in `[%%inject "zkapp_cmd_limit", zkapp_cmd_limit]`, but the code errored. Do you know why? Thanks.

Reply:
@deepthiskumar ^

Reply:
If you shared the error message then I can say for sure. But my guess is you are seeing a type mismatch where the expected type is `Option int` and injecting it here without creating a new value of option type will be inferred as int.  

You can do the following-

```
[%%ifndef zkapp_cmd_limit]

let zkapp_cmd_limit = None

[%%else]

[%%inject "zkapp_cmd_limit", zkapp_cmd_limit]

let zkapp_cmd_limit = Some zkapp_cmd_limit

[%%endif]

```